src/vt/runnable/runnable.cc

```diff
@@ -124,7 +124,16 @@ void RunnableNew::run() {
   {
     needs_time = contexts_.lb.needsTime();
   }
-  TimeType start_time = needs_time ? theSched()->getRecentTime() : TimeType{NAN};
+
+  auto getTime = [&]() {
+#if vt_check_enabled(trace_enabled)
+    if (theTrace()) return timing::getCurrentTime();
+    else
+#endif
+    return theSched()->getRecentTime();
```

Comment:
```suggestion
    {
      return theSched()->getRecentTime();
    }
```
Makes it more clear that there's a control flow statement preceding it.

This also better matches the style adopted for conditionally-compiled conditions.

Reply:
Fixed, thanks